src/test/java/com/google/api/codegen/testdata/discoveries/nodejs/nodejs_storage.v1.json.baseline

```diff
@@ -1571,10 +1592,19 @@ google.auth.getApplicationDefault(function(err, authClient) {
       // TODO: Add desired properties of the request body.
     },
 
-
-    // TODO: Add desired media content for upload. For more information, see:
+    // TODO: Use `media` to add desired media content for upload. For more information, see:
```

Comment:
On second look, this update no longer makes sense in https://github.com/googleapis/toolkit/pull/538, since the code is not commented out.